ROADMAP.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+The Bottlerocket roadmap is public and hosted as a GitHub Project at the following URL:
+
+https://github.com/orgs/bottlerocket-os/projects/18
+
+GitHub Projects allows for multiple views that represent different ways of looking at the same data. In the case of Bottlerocket, there are four views (represented as tabs):
```

Comment:
nit: in other MD docs we've followed a "one line per sentence" convention, which is a little nicer for people using terminal-based editors

Reply:
Not to my knowledge - is there a logical place for it to go?